index.js

```diff
@@ -37,11 +37,12 @@ const limit = Math.abs(cli.flags.limit);
 const chart = cli.flags.chart;
 const log = cli.flags.log;
 const minimal = cli.flags.minimal;
+const dangerous = cli.flags.dangerous;
 const options = { sortBy, limit, reverse, minimal, chart, log };
 
 (async () => {
 	// Init.
-	init(minimal);
+	init(dangerous);
```

Comment:
This is wrong. You remove minimal option all together.

Reply:
@ahmadawais : Sorry I did not get you. `minimal` option is not used inside the `init` module at all. That is the reason I removed it from the call.